pkg/db/sql_api.go

```diff
@@ -30,7 +48,9 @@ func WriteMsg(db *sql.DB, msg msgs.CAN) error {
 func TimeWindowedRead(db *sql.DB, canID uint16, from time.Time, to time.Time) ([]msgs.CAN, error) {
 	rows, err := db.Query(`
     SELECT
+      can.source,
       can.id,
```

Comment:
Is this the msg ID or raw CAN ID?

Reply:
Msg ID. I assume that we are using our network layer for CAN messages.